cdap-data-fabric/src/main/java/io/cdap/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetDataset.java

```diff
@@ -1059,25 +963,6 @@ private void disableExplore() {
   public static void fixPartitions(Transactional transactional, final String datasetName,
                                    boolean doDisable, final int partitionsPerTx, final boolean verbose) {
```

Comment:
`doDisable` parameter is not used anymore

Reply:
Removed